src/server_manager/web_app/gallery_app/main.ts

```diff
@@ -84,15 +93,15 @@ const GCP_BILLING_ACCOUNTS: gcp.BillingAccount[] =
 @customElement('outline-test-app')
 export class TestApp extends LitElement {
   @property({type: String}) dir = 'ltr';
-  @property({type: Function}) localize: (...args: string[]) => string;
+  @property({type: Function}) localize: (...args: string[]) => string = fakeLocalize;
+  @property({type: String}) language = 'es-419';  // Replaced asynchronously in the constructor.
```

Comment:
Perhaps use something invalid like `zz`?

Reply:
Done.  (Curious that `''` throws an exception but `'zz'` is allowed.)